### PR TITLE
doc: add docs

### PR DESCRIPTION
**Pull request by PaleBluDot**
A series of markdown files under the `docs/` directory that helps to explain the repository. 

- getting started
- commit
- commands
- pull request

### CODE REVIEW

**Comment by PaleBluDot** [comment]
@barsinister hold off until I get into to marge this request.  Have to make a change before we merge into main.

**Review by hellogoodbyte — APPROVED** [review]

**Comment by PaleBluDot** [comment]
@barsinister @hellogoodbyte this can now be reviewed and squashed once approved

**Review by barsinister — APPROVED** [review]

**Review by hellogoodbyte — APPROVED** [review]

**Pull request merged**
